.ci_scripts/start_docker.sh

```diff
@@ -8,6 +8,8 @@ case "$BUILD_TARGET" in
 	docker run -d --name vitasdk --workdir /build/git -v "${PWD}:/build/git" gnuton/vitasdk-docker:20190626 tail -f /dev/null
 	;;
 "switch")
-	docker run -d --name switchdev --workdir /build/git -v "${PWD}:/build/git" rsn8887/switchdev tail -f /dev/null
+	wget https://github.com/devkitPro/pacman/releases/download/devkitpro-pacman-1.0.1/devkitpro-pacman.deb
```

Comment:
A bit weird to install packages in a script file named `start_docker.sh`... Rename this file to `setup_environment.sh`?
Also: do we want to cache this file?

Reply:
Yes, I'm going to change the name when I get vita off docker as well :)

I'm not sure it is worth it to cache the file, as instead of it being fetched from github, it'll instead be fetched from the travis servers...